### DISCUSSION TITLE
Welcome to Giants_Software_USB_Beacon Discussions!

### DISCUSSION THREAD

**Discussion by Microgenital** [Microgenital/Giants_Software_USB_Beacon#1] [General]
## 👋 Welcome!
  Im trying Discussions as a place to where people can ask questions, hopefully i can answer them.
  * Ask questions you’re wondering about.
  * Share ideas.

  Have fun!

**Comment by M5trBl5tr** [comment]
Hallo
hab endlich die Möglichkeit gefunden die LS22 Rundumleuchte auch außerhalb des Spiels zu nutzen.
DANKE DANKE DANKE
Hab mir jedoch aus dem Script eine "exe" (Auto-Py-to-Exe) erzeugt um sie in Windows für andere Sachen zu gebrauchen.
Bsp optischer Wecker in Windows
Eventuell wäre das eine Erweiterung wert aufzunehmen. Vieleicht nach dem Schema
"Betriebsart-Anzahl der Durchläufe-Dauer.exe" -> Blink/Rotate-1-00h-00m-10s.exe
Das wäre für viele Casual Benutzer ein One-Click-Lösung da die Python installation weg fällt, und die Dateien portable sind, also ohne jegliche Installation funktionieren.
Falls du kein Interesse daran hast würd ich micht bereit erklären die "Konvertierung" zu übernehmen, natürlich mit dem Verweis des ursprünglichen Python-Scripts.

**Reply by Microgenital** [reply]
Hallo,

Nur zu, mach damit was du möchtest, für mich selbst ist das Projekt im moment abgeschlossen. Mein Ziel war es einfach nur die Leuchte auch außerhalb des Spiels zum laufen zu bekommen. 
Bitte gib mir bescheid wenn du schon etwas vorzeigen kannst. Würde gerne sehen was andere aus dem Beacon machen.

Grüße,
